pages/en/lb4/Decorators.md

```diff
@@ -0,0 +1,379 @@
+  share same configurations and properties.
+  You can use model decorators to define a model and its properties.
+
+#### Model Decorator
+
+  Syntax: `@model(definition?: ModelDefinitionSyntax)`
+
+  Model decorator is a Class decorator.
+  In LoopBack 4, we inherit the model definition format from LoopBack 3,
+  you can find it in [Model definition JSON file](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html).
+  For usage examples, see [Define Models](Repositories.html#define-models)
+
+  *Please note we will elaborate more about model and model definition in #Model.htm,*
+  *and replace the link above with LB4 link*
+
+  By using a model decorator, you can define a model as your repository's metadata,
+  and resolve your repository with Legacy Juggler that defautly comes with a set of CRUD operations bound to your model.
+  A use case can be found in section [Repository decorator](#repository-decorator)
+
+  Or you also have the flexibility to define a model without being resolved with Legacy Juggler.
+  This allows you to implement database access functions by yourself. 
+
+   ```ts
+  // Missing example here
+  // Will be provided in Model.htm
+  // refer to [example code](https://github.com/strongloop/loopback-next-example/blob/master/services/account-without-juggler/repositories/account/models/Account.ts)
+  ```
+
+#### Property Decorator
+
+  Syntax: `@property(definition: PropertyDefinition)`
+
+  It is a Class property decorator, defines the metadata for a Model property.
+  The format of property definitions can be found in [Property definitions](https://loopback.io/doc/en/lb2/Model-definition-JSON-file.html#properties) 
+
+  For usage examples, see [Define Models](Repositories.html#define-models)
+
+### Relation Decorators
+
+  Relation represents how you navigate from one model to other models.
+
+  *This feature has not yet been released in alpha form. Documentation will be* 
+  *added here as this feature progresses.*
+
+#### Relation Decorator
+
+  Syntax: `@relation`
+  
+  Register a general relation.
+
+#### Specfic Relation Decorator
+  
+  Syntax:
+
+  - `@belongsTo` 
+  - `@hasOne` 
+  - `@hasMany` 
+  - `@embedsOne` 
+  - `@embedsMany` 
+  - `@referencesOne` 
+  - `@referencesMany`
+
+  Register a specific relation
+
+### Repository Decorator
+
+  Syntax: `@repository(model: string | typeof Entity, dataSource?: string | juggler.DataSource)`
+
+  This decorator either injects an existing repository or creates a repository 
+  from a model and a datasource.
+
+  The injection example can be found in [Repository#controller-configuration](Repositories.html#controller-configuration)
+
+  To create a repository in a controller, you can define your model and datasource 
+  first, then import them in your controller file:
+
+  *Know more about how to create model and datasource, please see example in [Thinking in LoopBack](Thinking-in-LoopBack.htm#define-product-model-repository-and-data-source)*
```

Comment:
`To learn more about creating models and datasources, please see the example in...`